apps/lab/__tests__/e2e/test.cy.tsx

```diff
@@ -4,4 +4,10 @@ describe('First test', () => {
 
     cy.findByText("Tech lab").should('be.visible');
   });
+
+  it("Should check if the button exists", () => {
+    cy.visit('/');
+
+    cy.contains('Click Here').click();
```

Comment:
We should use accessible roles to verify UI elements. Something like this would be better:

```
const button = cy.findByRole('button', { name: 'Click Here' })

button.should('be.visible');
button.click();
```

Reply:
alright